docs/docs/notification-center/iframe-embed.md

```diff
@@ -174,12 +174,13 @@ novu.init(
-| `position.left`       | `string` \| `number` | Override the left position of the notification center drop down                                                      |
-| `theme`               | `object`             | Provide a custom theme for the notification center to use (for example see [above](#customizing-the-theme))          |
+| Parameter             | Type                 | Description                                                                                                                                                                                                           |
+| --------------------- | -------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `bellSelector`        | `string`             | A `class` or `id` of the notification bell in your UI. We will attach an event listener for it.                                                                                                                       |
+| `unseenBadgeSelector` | `string`             | A selector to the unseen count badge (the red dot) which Novu can use to populate in case unseen notifications exist                                                                                                  |
+| `backendUrl`          | `string`             | Custom API location in case of self-hosted version of Novu                                                                                                                                                            |
+| `socketUrl`           | `string`             | Custom WebSocket Service location in case of self-hosted version of Novu                                                                                                                                              |
+| `position.top`        | `string` \| `number` | Override the top position of the notification center drop down                                                                                                                                                        |
+| `position.left`       | `string` \| `number` | Override the left position of the notification center drop down                                                                                                                                                       |
+| `theme`               | `object`             | Provide a custom theme for the notification center to use (for example see [above](#customizing-the-theme))                                                                                                           |
+| `styles`              | `object`             | Provide a custom styles for the notification center. Please note that the limitation is that you can't use the style function as a prop to override the component styles. Read more about [styling](#custom-styling). |
```

Comment:
updated the docs about passing the `styles` prop in the `init` function argument